hazelcast/src/main/java/com/hazelcast/spi/impl/servicemanager/impl/ServiceManagerImpl.java

```diff
@@ -163,6 +164,7 @@ private void registerDefaultServices(ServicesConfig servicesConfig) {
         registerService(CRDTReplicationMigrationService.SERVICE_NAME, new CRDTReplicationMigrationService());
         registerService(DistributedScheduledExecutorService.SERVICE_NAME, new DistributedScheduledExecutorService());
         registerService(MetricsService.SERVICE_NAME, new MetricsService(nodeEngine));
+        registerService(JobEventService.SERVICE_NAME, new JobEventService(nodeEngine));
```

Comment:
it would be better if `JobEventService` was registered near `JetServiceBacked` and only if Jet is enabled